awpy/types.py

```diff
@@ -665,14 +665,81 @@ class RoundStatistics(TypedDict):
     players_killed: dict[Literal["CT", "T"], set[str]]
 
 
-class FrameMapControl(TypedDict):
-    """Type for the result of awpy.analytics.map_control.calc_map_control.
+# Type to represent tile id for navigation tiles.
+# TODO: Incorporate TileIdObject into rest of repo.
+TileIdObject = NewType("TileIdObject", int)
 
-    Holds map control values for each time for a certain frame
+# eturn type for awpy.analytics.map_control._bfs_helper.
+# Contains map control values for one team.
+# Maps TileIdObject to list of tile map control values.
+TeamMapControlValuesDict = NewType(
+    "TeamMapControlValuesDict", dict[TileIdObject, list[float]]
+)
+
+# Return type for awpy.analytics.map_control.graph_to_tile_neighbors
+# Maps TileIdObject to set of neighboring tiles.
+TileNeighborsDict = NewType("TileNeighborsDict", dict[TileIdObject, set[int]])
+
+
+class TileDistanceObject(TypedDict):
+    """Dataclass with data for map control tile distance calculations.
+
```

Comment:
Yup makes sense to me. Previously I thought the `@dataclass` was giving me a weird error but I was wrong. I forgot to uncomment these lines/add them for every relevant new class so this will be addressed in the next commit to the branch.